adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/AbstractVertxBasedMqttProtocolAdapter.java

```diff
@@ -301,17 +302,27 @@ final void handleEndpointConnection(final MqttEndpoint endpoint) {
         }
     }
 
-    private void handleConnectionRequestResult ( final MqttEndpoint endpoint, final AsyncResult<Device> done) {
+    private void handleConnectionRequestResult(final MqttEndpoint endpoint, final AsyncResult<Device> done) {
```

Comment:
can we find another name for parameter *done*? the type of the async result suggests that this is the result of the attempt to authenticate the device?